mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulHome.kt

```diff
@@ -20,8 +20,11 @@ private const val SocialRoute = "social"
 /** The route associated to the settings tab. */
 private const val SettingsRoute = "settings"
 
-/** The route associated to the game tab. */
-private const val GameRoute = "game"
+/** The route associated to the play tab. */
+private const val PlayRoute = "play"
+
+/** The route associated to new game button */
+const val NewGameRoute = "new_game"
```

Comment:
Maybe `GameRoute` is a better name, since the route doesn't describe that we created a new game but just on which screen we are.

Reply:
Agree